oldScripts/pl_to_spreadsheet.py

```diff
@@ -0,0 +1,102 @@
+# Author: Naveen Nathan
+import time, requests, datetime
+
+import os.path
+import json
+import pandas as pd
+from dotenv import load_dotenv
+"""
+from google.auth.transport.requests import Request
```

Comment:
Is there a reason this is commented out?

Reply:
I was planning to use these to write to sheets in the future. Perhaps I should remove it from this PR, though.